app/js/components/manageApps/ManageApps.jsx

```diff
@@ -487,6 +506,13 @@ export default class ManageApps extends React.Component {
                     isOpen={this.state.isOpen}
                     hideModal={this.hideModal}/>
                 ) : null}
+                {
+                  displayInvalidZip ? (
```

Comment:
Avoid rendering `null`, you can short-circuit this piece of code. I.e 
`displayInvalidZip && <InvalidZipUploadModal />`